src/services/ApiService.ts

```diff
@@ -761,3 +762,48 @@ export const getNftMetadataFromIpfs = async (cid: string): Promise<NftMetadata>
 
     return response.json();
 };
+
+export const getContractErrorMessage = async (network: Network, param: string, contractId: string) => {
+    const MAX_ATTEMPTS = 5;
+    const INTERVAL_MS = 400;
+    
+    for (let attempt = 1; attempt <= MAX_ATTEMPTS; attempt++) {
+        try {
+            let response: any = await new Promise((resolve, reject) => {
+                setTimeout(async () => {
+                    try {
+                        const result = await GET(network, `/transactions/${param}`);
+                        resolve(result);
+                    } catch (error) {
+                        reject(error);
+                    }
+                }, INTERVAL_MS);
+            });
+
+            if (response && response.transactions && response.transactions.length > 0) {
+                const consensusTimestamp = response.transactions[0].consensus_timestamp;
+
+                let contractResponse: any = await new Promise((resolve, reject) => {
+                    setTimeout(async () => {
+                        try {
+                            const result = await GET(network, `/contracts/${contractId}/results/${consensusTimestamp}`);
+                            resolve(result);
+                        } catch (error) {
+                            console.log(error);
+                            
+                            reject(error);
+                        }
+                    }, INTERVAL_MS);
+                });
+
+                return contractResponse?.error_message;
+            } else {
+                throw new Error("No transactions found in the response");
+            }
+        } catch (error) {
+            if (attempt === MAX_ATTEMPTS) {
+                throw new Error(`Failed to fetch contract error message after ${MAX_ATTEMPTS} attempts`);
```

Comment:
if we can't fetch error reason we need to skip this problem, and return to original flow  where CONTRACT_REVERT_EXECUTED without reason

Reply:
Fixed ✅